src/Project.php

```diff
@@ -9,18 +9,20 @@ class Project
     private string $id;
     private ?string $mlflowUri = '';
     private ?string $mlflowAbsSas = '';
+    private ?string $mlflowAbsConnectionString = '';
 
     private string $createdTimestamp;
     private string $updatedTimestamp;
 
     public static function fromArray(array $in): self
     {
-        $project = new Project();
-        $project->setId((string) $in['id']);
-        $project->setMLflowUri($in['mlflowUri'] ?? '');
-        $project->setMLflowAbsSas($in['mlflowAbsSas'] ?? '');
-        $project->setCreatedTimestamp($in['createdTimestamp']);
-        $project->setUpdatedTimestamp($in['updatedTimestamp'] ?? '');
+        $project = new self();
+        $project->id = (string) $in['id'];
+        $project->mlflowUri = $in['mlflowUri'] ?? '';
+        $project->mlflowAbsSas = $in['mlflowAbsSas'] ?? '';
+        $project->mlflowAbsConnectionString = $in['mlflowAbsConnectionString'] ?? '';
+        $project->createdTimestamp = $in['createdTimestamp'];
+        $project->updatedTimestamp = $in['updatedTimestamp'] ?? '';
```

Comment:
just curious why this change from setters?  (though I must say I prefer `new self()`)

Reply:
There is no setter for `mlflowAbsConnectionString` (it's generated on the API server side) and I wantend to be consitent.

Also it's not nessecary, the method is part of the class, so it has access to its private scope.